packages/grafana-ui/src/components/Tabs/Tab.tsx

```diff
@@ -53,32 +53,28 @@ export const Tab = React.forwardRef<HTMLElement, TabProps>(
 
     if (href) {
       return (
-        <div className={tabsStyles.item}>
```

Comment:
Can we remove the `item` styles then?

Reply:
This does make the tabs lose a little bit of padding, are we sure we are ok with that?

Reply:
I've tried to solve the focus issue 🤔. Similar issues appear over Grafana.  Maybe I can keep the div, and just set the role on that.